README.md

```diff
@@ -11,26 +11,31 @@ command line tools in Go featuring:
 - commands and subcommands with alias and prefix match support
 - flexible and permissive help system
 - dynamic shell completion for `bash`, `zsh`, `fish`, and `powershell`
-- `man` and markdown format documentation generation
-- input flags for simple types, slices of simple types, time, duration, and others
+- no dependencies except Go standard library
+- input flags for simple types, slices of simple types, time, duration, and
+  others
 - compound short flag support (`-a` `-b` `-c` can be shortened to `-abc`)
+- documentation generation in `man` and Markdown (supported via the
+  [`urfave/cli-docs`][urfave/cli-docs] module)
 - input lookup from:
   - environment variables
   - plain text files
-  - [structured file formats supported via the `urfave/cli-altsrc` package](https://github.com/urfave/cli-altsrc)
+  - structured file formats (supported via the
+    [`urfave/cli-altsrc`][urfave/cli-docs] module)
 
 ## Documentation
 
 See the hosted documentation website at <https://cli.urfave.org>. Contents of
 this website are built from the [`./docs`](./docs) directory.
 
-## Q&A
+## Support
 
-Please check the [Q&A discussions] or [ask a new question].
+Check the [Q&A discussions]. If you don't find answer to your question, [create
+a new discussion].
 
-## License
+If you found a bug or have a feature request, [create a new issue].
 
-See [`LICENSE`](./LICENSE).
+Please keep in mind that this project is run entirely by unpaid volunteers.
```

Comment:
That's a speculation. Maybe somebody is paid. I don't know.

Reply:
Hehe, I wish.

I removed "entirely", it sounded too strong. Or would you prefer to remove this line completely?